scripts/crons/cron_libraries.sh

```diff
@@ -69,12 +69,13 @@ do
   techo "Phase 1: end"
 done
 
-MANUAL_VERSION_OVERRIDE="47.0b1 47.0b2 47.0b3 47.0b4 47.0b5 47.0b6 47.0b7 47.0b8 47.0b9 47.0b99 48.0a2 49.0a1"
+#MANUAL_VERSION_OVERRIDE="47.0b1 47.0b2 47.0b3 47.0b4 47.0b5 47.0b6 47.0b7 47.0b8 47.0b9 47.0b99 48.0a2 49.0a1"
+VERSIONS=`./active-firefox-versions.py`
```

Comment:
SC2006: Use $(..) instead of legacy `..`. see: https://github.com/koalaman/shellcheck/wiki/Sc2006